server/src/main/java/org/apache/iotdb/db/metrics/ui/MetricsPage.java

```diff
@@ -0,0 +1,172 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.metrics.ui;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+import org.apache.iotdb.db.conf.IoTDBConstant;
+import org.apache.iotdb.db.metrics.server.SqlArgument;
+import org.apache.iotdb.service.rpc.thrift.TSExecuteStatementResp;
+
+import com.codahale.metrics.MetricRegistry;
+
+public class MetricsPage {
+
```

Comment:
Do we have some better way to make `html` as standalone templete rather than hard code them in java file ?

Reply:
IoTDB doesn't have special web UI, it uses Grafana. When it has web UI, the monitoring web page should be added.

Reply:
Ok, I'm going to try to turn the HTML code into an HTML file

Reply:
Well, there's some  information on the page about memory ,cpus and JVM